RELEASE-NOTES

```diff
@@ -102,6 +102,20 @@
    - support for hold invoices
    - support for onion messages
 
+ * Security:
+   - Mitigate against dust attacks; avoid spending from used addresses
```

Comment:
if you mean `config.WALLET_FREEZE_REUSED_ADDRESS_UTXOS`, note that it is disabled by default